src/ui/components/Notice/styles.scss

```diff
@@ -0,0 +1,97 @@
+@import "~core/css/inc/mixins";
+@import "~core/css/inc/vars";
+@import "~ui/css/vars";
+@import "~photon-colors/colors";
+
+.Notice {
+  border-radius: $border-radius-s;
+  display: grid;
+  font-size: $font-size-s;
+  grid-gap: 4px;
+  grid-template-columns: 20px auto;
+  line-height: 15px;
```

Comment:
`line-height` [should be in unitless measurements](https://css-tricks.com/almanac/properties/l/line-height/#article-header-id-0); I'm guessing this is probably around `1.2` but I'm not sure, you'll have to trial+error it 🙃